docs/artifact_repositories/nuget.md

```diff
@@ -0,0 +1,100 @@
+# Phylum NuGet Registry
+
+The Phylum NuGet registry is based on NuGet's [Server API].
```

Comment:
We should probably specify that we implement the v3 API. It's pretty obvious from the "v3" in the URL and the link to the v3 API documentation, but I think the v2 API is still widely supported and Artifactory might still call this "NuGet v3" as opposed to just "NuGet": https://jfrog.com/help/r/jfrog-artifactory-documentation/nuget-api-v3-registry-support

Reply:
But we do, that's why I've put the link there.